graphics/inkscape/Portfile

```diff
@@ -99,6 +99,10 @@ platform darwin {
     }
 }
 
+platform darwin 17 {
+    patchfiles-append libavoid-abs-ambiguous.diff
+}
+
```

Comment:
As this is a problem with the source not following C++ rules, the patch should be applied unconditionally.

Reply:
Fixed.